README.md

```diff
@@ -1042,66 +1042,66 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 *Libraries for parsing and manipulating texts.*
 
 * Specific Formats
```

Comment:
See, this title doesn't have a period either. Just pointing it out, nothing to change here.